docs/project/faq.md

```diff
@@ -486,36 +488,59 @@ other parts of LLVM will be advantageous, but not required.
+### When do we re-visit decisions or re-open discussions?
 
-When it comes to designing a language, there are lots of decisions where there
-are many workable options, and all of them will get the job done without serious
-issues. We need to pick _one_ in order to make progress, and no choice is going
-to appeal to everyone's preferences. We try to base the decision on consistent
-rationale, which can be found in the proposals which are linked throughout the
-design.
-
-We understand that many decisions won't be universally popular. Different
-developers weigh factors differently. However, we need to commit to a choice in
-order to make progress on the language design as a whole. This doesn't mean that
-they're _definitely_ right or set in stone, it just means we need to focus our
-time and energy on other issues.
+Once a decision is made through the [evolution process](evolution.md) the
+community should treat it as _firmly_ decided. This doesn't mean that the
+decision is _definitely_ right or set in stone, it just means we'd like the
+community to focus time and energy on other issues in the name of progress.
 
 Sometimes, it will be appropriate to revisit a decision; for example, when there
 is new information introduced, a new community joins Carbon, or there is an
-order of magnitude growth in the community. We will do this by filing an issue
-for leads with what new information is available, and we will reevaluate the
-choice.
+order of magnitude growth in the community. These cases are handled as new
+proposals through the [evolution process](evolution.md).
 
-For example, we're doing this with digit separators: we missed important
+For example, we have done this with digit separators: we missed important
 _domain_ conventions and had overly restricted where separators are allowed,
 [an issue was filed with the new information](https://github.com/carbon-language/carbon-lang/issues/1485),
 and we're fixing the choice.
 
-When there is a discussion about options or disagreement with a decision,
-comments saying "I like X" and "I don't like X" do not contribute to the
-conversation. You should use the reactions to signal this kind of sentiment, and
-we'll pay attention to those. Beyond that, we encourage folks to either shrug
-and let quirky syntax be quirky, or look for the objective use case that was
-clearly missed in the previous decision and makes the current state _sharply_
-problematic before pushing for change.
+See also the related questions
+[What happens when a decision was wrong?](#what-happens-when-a-decision-was-wrong),
+[How does Carbon make decisions?](#how-does-carbon-make-decisions) and
+[What can I do if I disagree with a design decision?](#what-can-i-do-if-i-disagree-with-a-design-decision).
+
+### What can I do if I disagree with a design decision?
+
+We invite you to give us constructive feedback. Some of Carbon's design
```

Comment:
```suggestion
The Carbon language is still young, and there are aspects of the language that
are provisional and expected to change. We invite you to give us constructive feedback. Some of Carbon's design
```

Maybe more appropriate up here? I'm uncertain.

Reply:
I ended up removing the "Carbon is still young" sentence since it didn't seem valuable in context.